index.js

```diff
@@ -49,10 +51,13 @@ try {
 	lr.on('end', function() {
 		for (const [key, value] of Object.entries(obj)) {
 			if (value.includes("FAIL") && value.includes("_test.go")) {
-				const parts = value.split("%0A")[1].trim().split(":");
-				const file = key.split("/").slice(0, -1).join("/") + "/" + parts[0];
-				const lineNumber = parts[1];
-				core.info(`::error file=${file},line=${lineNumber}::${value}`)
+				var result;
+				while ((result = regex.exec(value)) !== null) {
```

Comment:
why do you use `while` here?
if the condition was met, we will be stuck for good in the while-loop.

Reply:
Got the while loop from the [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec) to loop through each regex match, which in this case would be each assert failure. 

The condition should always be met when there are no more regex matches.